desire/index.html

```diff
@@ -186,12 +187,12 @@ <h3>Desire Paths</h3><hr>
 
 function addDateToAttribution(){
   const request = new XMLHttpRequest()
-  request.open('GET','https://api.mapbox.com/tilesets/v1/bikeottawa?access_token=sk.eyJ1IjoiYmlrZW90dGF3YSIsImEiOiJjamdyYTJmN2EwMmtoMzJwc3JxM2hoZ3ozIn0.YB1JNmncsPvktmgGU_zK8A')
+  request.open('GET','https://api.mapbox.com/tilesets/v1/' + mapbox_username + '?access_token=' + mapbox_token)

```

Comment:
Shouldn't this be the secret token?
```suggestion
  request.open('GET','https://api.mapbox.com/tilesets/v1/' + mapbox_username + '?access_token=' + mapbox_secret_token)
```

Reply:
Yes, indeed. That looks like it should be. Sorry about that.